internal/fptest/portspeed.go

```diff
@@ -0,0 +1,43 @@
+// 2022 Nokia.  This code is a Contribution to OpenConfig Feature Profiles
```

Comment:
Use the license generated by addlicense?

```
go install github.com/google/addlicense@latest
addlicense -l apache -s -c "Nokia, Google LLC" .
```

Reply:
Updated license by addlicense.